nixos/tests/kerberos/heimdal.nix

```diff
@@ -23,31 +23,21 @@ import ../make-test.nix ({pkgs, ...}: {
   };
 
   testScript = ''
-    $machine->start;
+    machine.succeed(
+        "kadmin -l init --realm-max-ticket-life='8 day' --realm-max-renewable-life='10 day' FOO.BAR"
+    )
 
-    $machine->succeed(
-      "kadmin -l init --realm-max-ticket-life='8 day' \\
-       --realm-max-renewable-life='10 day' FOO.BAR"
-    );
+    machine.succeed("systemctl restart kadmind.service kdc.service")
+    machine.wait_for_unit("kadmind.service")
+    machine.wait_for_unit("kdc.service")
+    machine.wait_for_unit("kpasswdd.service")
 
-    $machine->succeed("systemctl restart kadmind.service kdc.service");
-    $machine->waitForUnit("kadmind.service");
-    $machine->waitForUnit("kdc.service");
-    $machine->waitForUnit("kpasswdd.service");
-
-    $machine->succeed(
-      "kadmin -l add --password=admin_pw --use-defaults admin"
-    );
-    $machine->succeed(
-      "kadmin -l ext_keytab --keytab=admin.keytab admin"
-    );
-    $machine->succeed(
-      "kadmin -p admin -K admin.keytab add --password=alice_pw --use-defaults \\
-       alice"
-    );
-    $machine->succeed(
-      "kadmin -l ext_keytab --keytab=alice.keytab alice"
-    );
-    $machine->succeed("kinit -kt alice.keytab alice");
+    machine.succeed("kadmin -l add --password=admin_pw --use-defaults admin")
+    machine.succeed("kadmin -l ext_keytab --keytab=admin.keytab admin")
+    machine.succeed(
+        "kadmin -p admin -K admin.keytab add --password=alice_pw --use-defaults alice"
+    )
+    machine.succeed("kadmin -l ext_keytab --keytab=alice.keytab alice")
+    machine.succeed("kinit -kt alice.keytab alice")
```

Comment:
the whole test seems to be one big test case and not semantically unrelated test cases where individual output strings are used. Would anything speak against this slick implementation?:

```python
     machine.succeed(
        "kadmin -l init --realm-max-ticket-life='8 day' --realm-max-renewable-life='10 day' FOO.BAR",
        "systemctl restart kadmind.service kdc.service"
    )

    for unit in "kadmind.service", "kdc.service", "kpasswdd.service":
        machine.wait_for_unit(unit)

    machine.succeed(
        "kadmin -l add --password=admin_pw --use-defaults admin",
        "kadmin -l ext_keytab --keytab=admin.keytab admin",
        "kadmin -p admin -K admin.keytab add --password=alice_pw --use-defaults alice",
        "kadmin -l ext_keytab --keytab=alice.keytab alice",
        "kinit -kt alice.keytab alice"
    )
```

Reply:
This is just a minor style suggestion. Not a blocker.

Reply:
I didn't know `machine.succeed` takes multiple arguments. Done.